browser/src/control/Control.Command.js

```diff
@@ -81,19 +75,13 @@ L.Map.include({
 		});
 		message.push('</ul>', '</div>', '<div>');
 
-		vex.dialog.confirm({
-			unsafeMessage: message.join(''),
-			showCloseButton: false,
-			contentClassName: 'vex-content vex-locking ' + lockingOnMobile,
-			callback: function (value) {
-				if (value)
-					window.open(that.Locking.unlockLink, '_blank');
-			},
-			buttons: [
-				$.extend({}, vex.dialog.buttons.NO, { text: _('Cancel') }),
-				$.extend({}, vex.dialog.buttons.YES, { text: _('Unlock') })
-			]
-		});
+		message = message.join();
+
+		this.uiManager.showInfoModal('unlock-features-popup', this.Locking.unlockTitle, ' ', ' ', _('Unlock'), function() {
```

Comment:
what about cancel button? maybe we should use showConfirmModal ?

Reply:
I guess there wasn't a cancel button in the initial implementation. Maybe we can ask @pedropintosilva ?

Reply:
there was, see 5 lines above. there is "Cancel" and "Unlock" button